### DISCUSSION TITLE
Not able to install on Linux RHEL 8 and 9

### DISCUSSION THREAD

**Discussion by kamranattari4192** [dotnet/aspnetcore#49070] [Q&A]
Hi,

I am trying to install on RHEL 8.6 and 9.1 but it is failing. On both systems when I run following command:
sudo dnf install aspnetcore-runtime-7.0

It gives following error:

Updating Subscription Management repositories.
Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)                                                                                                                          12 kB/s | 4.1 kB     00:00
Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)                                                                                                                             11 kB/s | 4.1 kB     00:00
No match for argument: aspnetcore-runtime-7.0
Error: Unable to find a match: aspnetcore-runtime-7.0

**Comment by gfoidl** [comment]
@danmoseley this should be converted to an issue in https://github.com/dotnet/core?
(I don't own sufficient rights to do so)

Organisational side question: is there a team we (community triagers) can tag in such cases?
